code/src/components/Album.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const Album = (props) => {
+ const {images, artists , name} = props.items
```

Comment:
great to see deconstruction here! You could make this code even simpler like this:
const Album = ({images, artists, name}) =>{}